src/EVM.hs

```diff
@@ -1799,26 +1797,26 @@ cheatActions = Map.fromList
       \sig input -> case decodeStaticArgs 0 1 input of
         [addr]  -> case wordToAddr addr of
           Just a -> do
-            assign (#config % #overrideCaller) (Just a)
             doStop
```

Comment:
Can you add a comment explaining why doStop is done first? I imagine this lets you edit the parent's `#state` which is the one who should hold the prank, but it's non-obvious.

Reply:
This has been fixed now :)